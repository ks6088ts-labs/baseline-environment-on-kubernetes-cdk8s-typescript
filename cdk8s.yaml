language: typescript
app: npx ts-node bin/main.ts
imports:
  - k8s
