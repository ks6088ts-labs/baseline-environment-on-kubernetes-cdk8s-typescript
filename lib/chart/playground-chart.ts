import { Construct } from 'constructs';
import { Chart, ChartProps } from 'cdk8s';

export interface PlaygroundChartProps extends ChartProps {}

export const devPlaygroundChartProps: PlaygroundChartProps = {};
export const prodPlaygroundChartProps: PlaygroundChartProps = {};

export class PlaygroundChart extends Chart {
  constructor(scope: Construct, id: string, props: PlaygroundChartProps = {}) {
    super(scope, id, props);
  }
}
