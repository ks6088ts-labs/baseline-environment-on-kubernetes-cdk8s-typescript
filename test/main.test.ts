import { PlaygroundChart } from "../lib/chart/playground-chart";
import { Testing } from "cdk8s";

describe("PlaygroundChart", () => {
  test("PlaygroundChart", () => {
    const app = Testing.app();
    const chart = new PlaygroundChart(app, "playground-chart");
    const results = Testing.synth(chart);
    expect(results).toMatchSnapshot();
  });
});
