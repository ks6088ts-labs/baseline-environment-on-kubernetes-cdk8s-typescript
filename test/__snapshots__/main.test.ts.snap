// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaygroundChart PlaygroundChart 1`] = `[]`;
